content/en/blog/2024/go-opentelemetry-io.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Planned Migration for go.opentelemetry.io
+linkTitle: Planned migration for go.opentelemetry.io
+date: 2024-08-23
+cSpell:ignore: Yahn
+author: >-
+  [Mike Dame](https://github.com/damemi) (Google),
+# canonical_url: http://somewhere.else/ # TODO: if this blog post has been posted somewhere else already, uncomment & provide the canonical URL here.
+body_class: otel-with-contributions-from # TODO: remove this line if there are no secondary contributing authors
+issue: 5086
+sig: Go SIG
+---
+
+With contributions from Tyler Yahn and Austin Parker.
+
+## TL;DR
+
+The app that serves requests to [go.opentelemetry.io] will be migrating to a new
+host on September 5, 2024 at approximately 11:00AM Eastern Time. There are no
+changes required from you. We are making this announcement because it is
```

Comment:
```suggestion
host on September 5, 2024 at approximately 11:00AM Eastern Time. **There are no
changes required from you**. We are making this announcement because it is
```